packages/orbit-components/src/Popover/index.js

```diff
@@ -135,6 +136,7 @@ const Popover = ({
         <Portal renderInto="popovers">
           <PopoverContentWrapper
             shown={shown}
+            offset={offset}
```

Comment:
This way people would always have to specify both `top` and `left`. Do we want to allow them to omit one of those, and let it default to `0`? In that case I think you could do something like this instead:

```suggestion
            offset={{ top: 0, left: 0, ...offset }}
```